sensu/gohandler_test.go

```diff
@@ -57,7 +59,29 @@ var (
 		Usage:     "Third argument",
 	}
 
-	defaultCmdLineArgs = []string{"--arg1", "value-arg1", "--arg2", "7531", "--arg3=false"}
+	defaultOption4 = PluginConfigOption{
+		Argument:  "arg4",
+		Default:   []string{},
```

Comment:
What happens if the default is nil? Should add another test case for that.

Reply:
Added Default nil tests for all options.